pir/fast_pir/fast_pir_client.hpp

```diff
@@ -32,6 +40,7 @@ class FastPIRClient {
   using pir_query_t =
       FastPIRQuery<seal::Serializable<seal::Ciphertext>, Galois_string>;
   using pir_answer_t = FastPIRAnswer;
+  using pir_map = std::map<pir_index_t, keys>;
 
   // TODO: fix this absolute mess of initializers???
```

Comment:
let's remove this comment. we have fixed it now :)

Reply:
resolved.